lib/ButtonNew/ButtonIcon/ButtonIconDanger.js

```diff
@@ -17,17 +17,22 @@ function ButtonIconDanger({
   size,
   ...rest
 }) {
-  const classes = cx(
+  const nonDisabledClasses = cx(
     'p-0 bg-white',
     'inherit-color-icon text-red-primary',
     'items-center justify-center',
-    'border-solid border-transparent',
-    'focus:border-red-primary focus:shadow-red-focus-sm',
-    className,
+    'focus:shadow-button-icon-danger-focus',
     {
-      'hover:bg-red-95 active:bg-red-90': !disabled && !active,
-      'opacity-50': disabled,
+      'hover:bg-red-95 active:bg-red-90': !active,
       'bg-red-90': active
+    }
+  );
+
+  const classes = cx(
+    className,
+    {
+      [nonDisabledClasses]: !disabled,
```

Comment:
oOooh, is that because `nonDisabledClasses` has already evaluated as a string by now?